### DISCUSSION TITLE
[bug] Get bnb balance format error, which may occur in some cases

### DISCUSSION THREAD

**Discussion by phpmac** [reown-com/appkit#2397] [Q&A]
### Link to minimal reproducible example

no

### Summary

获得BNB余额出现错误，测试发现demo也会有这个问题，在 [web3modal](https://web3modal.com/) 访问会测试出来，用的是公开私钥的eth钱包 `0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266` 私钥是 `ac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80`

下面是截图：

<img width="987" alt="截屏2024-05-30 18 55 43" src="https://github.com/WalletConnect/web3modal/assets/86544347/9ce54df3-c71e-4be4-b93c-33e2f1b5818e">

### List of related npm package versions

最新版本最新测试在线demo

**Comment by glitch-txs** [comment]
Do you know if this number is ETH or other coin Balance?
